test/typesArray.test.js

```
@@ -0,0 +1,185 @@
+        type: 'array',
+        items: {
+          type: ['string', 'number']
+        }
+      }
+    }
+  }
+  const stringify = build(schema)
+
+  try {
+    const value = stringify({
+      arrayOfStringsAndNumbers: ['string1', 'string2']
+    })
+    t.is(value, '{"arrayOfStringsAndNumbers":["string1","string2"]}')
+  } catch (e) {
+    console.log(e)
+    t.fail()
+  }
+
+  try {
+    const value = stringify({
+      arrayOfStringsAndNumbers: [42, 7]
+    })
+    t.is(value, '{"arrayOfStringsAndNumbers":[42,7]}')
+  } catch (e) {
+    t.fail()
+  }
+
+  try {
+    const value = stringify({
+      arrayOfStringsAndNumbers: ['string1', 42, 7, 'string2']
+    })
+    t.is(value, '{"arrayOfStringsAndNumbers":["string1",42,7,"string2"]}')
+  } catch (e) {
+    t.fail()
+  }
+})
+
+// TODO: Multiple types inside tuples are currently unsupported
+// test('object with tuple of multiple types', (t) => {
+//   t.plan(2)
+
+//   const schema = {
+//     title: 'object with array of multiple types',
+//     type: 'object',
+//     properties: {
+//       fixedTupleOfStringsAndNumbers: {
+//         type: 'array',
+//         items: [
+//           {
+//             type: 'string'
+//           },
+//           {
+//             type: 'number'
+//           },
+//           {
+//             type: ['string', 'number']
+//           }
+//         ]
+//       }
+//     }
+//   }
+//   const stringify = build(schema)
+
+//   try {
+//     const value = stringify({
+//       fixedTupleOfStringsAndNumbers: ['string1', 42, 7]
+//     })
+//     t.is(value, '{"fixedTupleOfStringsAndNumbers":["string1",42,7]}')
+//   } catch (e) {
+//     console.log(e)
+//     t.fail()
+//   }
+
+//   try {
+//     const value = stringify({
+//       fixedTupleOfStringsAndNumbers: ['string1', 42, 7]
+//     })
+//     t.is(value, '{"fixedTupleOfStringsAndNumbers":["string1",42,"string2"]}')
+//   } catch (e) {
+//     console.log(e)
+//     t.fail()
+//   }
+// })
```

Do you think it would be feasible to also implement this feature? If not, we should at least throw a nice error that this is not supported and test that.